vector/src/main/codegen/templates/UnionVector.java

```diff
@@ -306,6 +307,9 @@ public StructVector getStruct() {
 
   public ${name}Vector get${name}Vector(String name) {
     if (${uncappedName}Vector == null) {
+      ${uncappedName}Vector = internalStruct.getChild(fieldName(MinorType.${name?upper_case}), ${name}Vector.class);
```

Comment:
Should we apply to only child, or also on parameterized types ?

Reply:
I'm not sure, I'm submitting a change someone else made in our repo. Would it be any different if used the parameterized type instead?